package.json

```diff
@@ -58,5 +58,8 @@
       "eslint --fix",
       "prettier --write"
     ]
+  },
+  "dependencies": {
+    "glob": "^10.2.3"
```

Comment:
**question:** is it possible to get by without this dependency in the root of the project? It's not a super big one but if it's at the root then it'll be installed for every build of Reliability Kit regardless of whether we're actually running the script or not, which is a little bit of an overhead.